server/backends/invocationdb/invocationdb.go

```diff
@@ -61,6 +63,35 @@ func (d *InvocationDB) InsertOrUpdateInvocation(ctx context.Context, ti *tables.
 	})
 }
 
+func (d *InvocationDB) UpdateInvocationACL(ctx context.Context, invocationID string, acl *aclpb.ACL) error {
+	p, err := perms.ToPerms(acl)
```

Comment:
I think perms.FromACL(acl) would be a clearer name here. wdyt?

Reply:
SG, Done